### DISCUSSION TITLE
When 1 battery in parallel packs reaches 100% charging stops

### DISCUSSION THREAD

**Discussion by CaptKrisp** [Louisvdw/dbus-serialbattery#697] [Q&A]
I have 2 packs in parallel.  Both use Daly BMSs. Battery Aggregator is installed. DVCC is enabled.
When 1 of the packs reaches 100% SoC, but not both, the SmartSolar charge controller is commanded to stop charging.
When DVCC is disabled the SmartSolar charge controller begins charging again.

I have another system, also with 2 packs in parallel, and have _not_ noticed this issue in that system.
Both systems have the same VenusOS version as well as same same packages installed.

Any thoughts for the source of this issue?

**Comment by mr-manuel** [comment]
Did you read the documentation? Use Venus OS 3.0 and select the battery aggregator as battery monitor and BMS (under DVCC).

**Comment by CaptKrisp** [comment]
Venus OS 3.10 is installed.  Battery Aggregator is installed and selected as Battery Monitor under System Setup.

When Battery Aggregator is manually selected as Battery Monitor under System Setup, DVCC auto-selects Battery Aggregator.
When Automatic is selected as Battery Monitor under System Setup one of the physical BMSs is selected and DVCC follows that selection.

Could the problem be using VenusOS RC 3.10 instead of release 3.0?  Or perhaps any other ideas?
